### ISSUE TITLE
Symbiotic Germs game crash

### ISSUE THREAD

**Issue by wildejackson** [pether-pg/ONI_Mods_byPether#38]
Currently there is a crash whenever a slickster of any type excretes resources. I posted the stacktrace below something about incorrectly referencing the element slicksters poop out on innit. Note: I tried this both with and without a slickster morph mod as well which had exactly the same crash so I dont think theres incompatibility issues there)

[06:52:05.483] [1] [INFO] [Vanilla + DLC] Symbiotic Germs's assembly declared the method SymbioticGerms.SymbioticGerms_Patches+CreatureCalorieMonitor_Poop_Patch:Prefix in the stack trace, adding to referenced mods list
```
  [DebugConsole][1]: Error[Error]: Exception in: (Oilfloater).SameSpotPoopStates.root.behaviourcomplete.BehaviourComplete(Poop)
System.MissingFieldException: Field 'Element.idx' not found.

  at SymbioticGerms.SymbioticGerms_Patches+CreatureCalorieMonitor_Poop_Patch.Prefix (CreatureCalorieMonitor+Instance __instance) [0x0002f] in <06e04c98a3aa4c708b3b502ed06029e7>:0

  at (wrapper dynamic-method) CreatureCalorieMonitor+Instance.CreatureCalorieMonitor+Instance.Poop_Patch1(CreatureCalorieMonitor/Instance)

  at CreatureCalorieMonitor+<>c.<InitializeStates>b__7_1 (CreatureCalorieMonitor+Instance smi) [0x00000] in <322ab64cfe564736a3a4cd0cabcbbf3c>:0

  at GameStateMachine`4+State+<>c__DisplayClass56_0[StateMachineType,StateMachineInstanceType,MasterType,DefType].<ToggleBehaviour>b__3 (StateMachineInstanceType smi, System.Object data) [0x00013] in <322ab64cfe564736a3a4cd0cabcbbf3c>:0

  at GameStateMachine`4+GameEvent+<>c__DisplayClass6_0[StateMachineType,StateMachineInstanceType,MasterType,DefType].<Subscribe>b__0 (System.Object d) [0x00008] in <322ab64cfe564736a3a4cd0cabcbbf3c>:0

  at EventSystem.Trigger (UnityEngine.GameObject go, System.Int32 hash, System.Object data) [0x000a7] in <1b5c2ff4f53d4a48b7159ed84d905978>:0

  at KMonoBehaviour.Trigger (System.Int32 hash, System.Object data) [0x00026] in <1b5c2ff4f53d4a48b7159ed84d905978>:0

  at StateMachine+Instance.Trigger (System.Int32 hash, System.Object data) [0x0000b] in <322ab64cfe564736a3a4cd0cabcbbf3c>:0

  at GameStateMachine`4+State+<>c__DisplayClass55_0[StateMachineType,StateMachineInstanceType,MasterType,DefType].<BehaviourComplete>b__1 (StateMachineInstanceType smi) [0x00000] in <322ab64cfe564736a3a4cd0cabcbbf3c>:0

  at StateMachine`4+GenericInstance[StateMachineType,StateMachineInstanceType,MasterType,DefType].ExecuteActions (StateMachine`4+State[StateMachineType,StateMachineInstanceType,MasterType,DefType] state, System.Collections.Generic.List`1[T] actions) [0x00048] in <322ab64cfe564736a3a4cd0cabcbbf3c>:0
```

**Comment by pether-pg** [comment]
Thanks for the report, it looks that recent game update broke some things and I missed that.
I will focus on fixing that, but it may take a moment. Until that, if looks like you will need to disable the mod for a moment

**Comment by pether-pg** [comment]
issue fixed in 2022.10.21.2046
please make sure your mod is updated correctly and have fun :)
